client/services/AuthService.tsx

```diff
@@ -83,8 +83,12 @@ const AuthService = ({ children }) => {
       })
       const data = await res.json()
       if (data.status === 'Success') {
-        localStorage.setItem('TOKEN', data.token)
-        localStorage.setItem('ACCOUNT', req.email)
+        if (typeof window !== 'undefined') {
```

Comment:
kept getting localStorage undefined, tried this as a work around, didn't work...